src/modules/authentication/components/Login/Login.tsx

```diff
@@ -1,7 +1,121 @@
+import styles from "./Login.module.css";
+import imgLogo from "./../../../../assets/images/PMSLogo.svg";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import { AUTH_URL, axiosInstanceUrl } from "../../../../services/EndPoints";
+import { useState } from "react";
+
+interface loginForm {
+  email: string;
+  password: string;
+}
 export default function Login() {
+  const navigate = useNavigate()
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+  } = useForm<loginForm>();
+
+  const onSubmit = async (data: loginForm) => {
+    try {
+      const response = await axiosInstanceUrl.post(AUTH_URL.LOGIN, data);
+      console.log(response);
+      navigate("/dashboard")
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
   return (
-    <div>
-      <h1>Login</h1>
+    <div className={`${styles["containerImg"]}`}>
+      <div>
+        <img src={imgLogo} alt="" />
+      </div>
+      <div className={`${styles["fromContainer"]} col-lg-6 col-md-8 mt-3`}>
+        <p className="">welcome to PMS</p>
+        <h1>
+          {" "}
+          <u>L</u>ogin
+        </h1>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className={`${styles["input-group"]} mt-5 mb-3`}>
+            <input
+              type="text"
+              className={`${styles["inbut"]} form-control`}
+              placeholder="Email"
+              aria-label="email"
+              aria-describedby="basic-addon1"
+              {...register("email", {
+                required: "Email is required",
+                pattern: {
+                  value: /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g,
```

Comment:
We can extract the validation logic to be able to re-use it.

Reply:
also we did not have a file for the validation in the project file structure